### PR TITLE
docs: various fixes

### PR DESCRIPTION
**Pull request by TomJGooding**
**Please review the following checklist.**

- [ ] Docstrings on all new or modified functions / classes 
- [x] Updated documentation
- [ ] Updated CHANGELOG.md (where appropriate)

### CODE REVIEW

File: src/textual/dom.py [hunk 1]
```diff
@@ -514,7 +514,7 @@ def get_component_styles(self, *names: str) -> RenderStyles:
         """Get a "component" styles object (must be defined in COMPONENT_CLASSES classvar).
 
         Args:
-            name: Name of the component.
+            names: Names of the components.
```
Comment (TomJGooding) [inline src/textual/dom.py:517]: This whole docstring for `get_component_styles` might need updating to reflect recent changes?

File: src/textual/widget.py [hunk 2]
```diff
@@ -806,7 +806,7 @@ def get_component_rich_style(self, *names: str, partial: bool = False) -> Style:
         """Get a *Rich* style for a component.
 
         Args:
-            name: Name of component.
+            names: Names of components.
```
Comment (TomJGooding) [inline src/textual/widget.py:809]: This whole docstring for `get_component_styles` might need updating to reflect recent changes?

File: src/textual/dom.py (hunk 1 above)
Comment (willmcgugan) [reply src/textual/dom.py:517]: Yep, noted.

**Review by willmcgugan — APPROVED** [review]
Thanks Tom

**Pull request merged**